component/loopback/loopback_fs.go

```diff
@@ -208,6 +208,11 @@ func (lfs *LoopbackFS) RenameDir(options internal.RenameDirOptions) error {
 
 func (lfs *LoopbackFS) CreateFile(options internal.CreateFileOptions) (*handlemap.Handle, error) {
 	log.Trace("LoopbackFS::CreateFile : name=%s", options.Name)
+
+	if options.Name == "FailThis" {
```

Comment:
Any reason for this check?

Reply:
This is to run some negative test cases.